package.json

```diff
@@ -33,6 +33,7 @@
     "babel-preset-env": "^1.2.2",
     "babel-register": "^6.24.1",
     "babelify": "^7.3.0",
+    "bootstrap-sass": "^3.3.7",
```

Comment:
Do we still need `"bootstrap": "3.3.7"` in the dependencies?

Doesn’t ship `bootstrap-sass` all we need?